modules/ROOT/pages/upgrade-custom-policy-mule3.adoc

```diff
@@ -0,0 +1,35 @@
+= Upgrade Custom Policies for Mule 3
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In API Manager, you can upgrade the version of your custom policies for your Mule 3 applications.
```

Comment:
Let's add what problem are we solving with this
https://docs.google.com/document/d/1RQZr_kLzyGyqItMzjyrBT1-0_H5plaNf5M-_QCoyzvQ/edit#heading=h.6ek3exazel76

In Api Manager you can change the custom policy version without leaving your API insecure.